### ISSUE TITLE
Liking of post removes like on comment

### ISSUE THREAD

**Issue by Quix0r** [friendica/friendica#5075]
### Expected behavior
The like on comment remains and the like on post is being added.

### Actual behavior
On of many examples is [here](https://f.haeder.net/display/f87d4b91143144d2bfa4a0ec452dcfe8d333194a) when I try to like the post, the like from the comment by Hiker is being removed.

### Steps to reproduce the problem
- cross-fediverse post/comment construct
- like first comment
- then like post

### Friendica version you encountered the problem
`develop`

### Friendica source (git, zip)
`git`

### PHP version
````
PHP 7.0.30-1~dotdeb+8.1 (cli) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.30-1~dotdeb+8.1, Copyright (c) 1999-2017, by Zend Technologies
    with Xdebug v2.5.5, Copyright (c) 2002-2017, by Derick Rethans
````
### SQL version
`mysql  Ver 15.1 Distrib 10.1.26-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2`

**Comment by MrPetovan** [comment]
I've been liking posts and comments left and right yesterday and I couldn't reproduce this. However, sometimes the `update_network` post refresh doesn't show all the likes, can you please check on the post display page that likes are indeed missing?

**Comment by Quix0r** [comment]
Yes, there are really gone. Repeatable and gone. Which query should I perform so you can see the actual data? I may `x` out private data (I guess none is there).
